notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -14,121 +14,93 @@
    "metadata": {},
    "source": [
     "# Run Notebook on AML\n",
```

Comment:
What about "Submit an existing notebook to AzureML"

Reply:
Changed title line to "Submit an Existing Notebook to AzureML"